project3.0/index.html

```diff
@@ -5,57 +5,87 @@
+    <link href="https://fonts.googleapis.com/css2?family=Heebo:wght@100..900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
 </head>
 
 <body>
     <header class="header">
-        <div class="container">
+        <div class="container-fluid">
             <div class="header-wrapper">
-                <ul>
-                    <li><a href="">Works</a></li>
-                    <li><a href="">Blog</a></li>
-                    <li><a href="">Contact</a></li>
+                <div class="hamburger" onclick="toggleMenu()">
+                    <div></div>
+                    <div></div>
+                    <div></div>
+                </div>
+                <ul class="menu" id="menu">
+                    <li><a href="#">Works</a></li>
+                    <li><a href="#">Blog</a></li>
+                    <li><a href="#">Contact</a></li>
                 </ul>
+                <script>
+                    function toggleMenu() {
+                        var menu = document.getElementById('menu');
+                        var hamburger = document.querySelector('.hamburger');
+                        var content = document.getElementById('content');
+                        menu.classList.toggle('active');
+                        hamburger.classList.toggle('active');
+                        content.classList.toggle('shifted');
+                    } 
+                </script>
             </div>
         </div>
     </header>
-    <section class="section-info">
+    <section class="section-info" id="content">
         <div class="container">
             <div class="section-info-wrapper">
-                <h1>
-                    Hi, I am John,<br>Creative Technologist
-                </h1>
-                <p>
-                    Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat
-                    duis enim velit mollit. Exercitation veniam consequat sunt nostrud amet.
-                </p>
-                <div>
-                    <img src="img/foto.png" alt="foto">
+                <div class="info-left">
+                    <h1>
+                        Hi, I am John,<br>Creative Technologist
+                    </h1>
+                    <p>
+                        Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia
+                        consequat
+                        duis enim velit mollit. Exercitation veniam consequat sunt nostrud amet.
+                    </p>
+                    <div class="info-left-button">
+                        <button>Download Resume</button>
+                    </div>
                 </div>
-                <div>
-                    <button>Download Resume</button>
+                <div class="info-right">
+                    <div class="img-foto">
+                        <img src="img/foto.png" alt="foto">
```

Comment:
Change alt tag, usually write what you see on the image. Like: "Photo of the smiling person", "Picture of the red cat" etc.